featurebyte/schema/target_table.py

```diff
@@ -23,9 +23,9 @@ class TargetTableCreate(FeatureOrTargetTableCreate):
     """
 
     serving_names_mapping: Optional[Dict[str, str]]
-    target_id: Optional[PydanticObjectId]
-    graph: QueryGraph
-    node_names: List[StrictStr]
+    target_id: PydanticObjectId
+    graph: Optional[QueryGraph]
+    node_names: Optional[List[StrictStr]]
     request_input: ObservationInput
     context_id: Optional[PydanticObjectId]
     skip_entity_validation_checks: bool = Field(default=False)
```

Comment:
Could consider to add a root validator here to check that both `graph` and `node_names` should either co-exist or  both disappear.

Reply:
What if `target_id`, `graph` and `node_names` all are provided? Should we load the target graph & combine with the given graph?

Reply:
Is it possible to load more than 1 `target_id`? Curious why we use plural `node_names` but use singular `target_id`?

Reply:
If more than 1 `target_id` is supported, we could use the following ways to combine target graphs & nodes:
https://github.com/featurebyte/featurebyte/blob/83272c5327d7880fe9c4082a2d03d3deaaf651e0/featurebyte/query_graph/pruning_util.py#L15-L33